plugin/filetypes.lua

```diff
@@ -29,6 +29,7 @@ local filetypes = {
   ssh_config = { 'sshconfig' },
   starlark = { 'bzl' },
   surface = { 'sface' },
+  sourcepawn = { 'sp', 'inc' },
```

Comment:
This is for filetypes, not extensions.
If sourcepawn does not currently have a filetype, it should be added to Vim.

```suggestion
```